src/components/RightSidebar/SharedItems/SharedItems.vue

```diff
@@ -35,6 +35,20 @@
 				<DeckCard :wide="true"
 					v-bind="item.messageParameters.object" />
 			</div>
+			<template v-else-if="type === 'other'">
+				<div :key="item.id"
+					class="shared-items__other">
+					<a v-if="item.messageParameters.object.link "
+						:href="item.messageParameters.object.link"
+						target="_blank">
+						{{ item.messageParameters.object.name }}
+					</a>
+					<p v-else
+						:key="item.id">
```

Comment:
Since the key is on the div now we can remove it here to deduplicate the keys?
```suggestion
					<p v-else>
```